mazda/Makefile

```diff
@@ -74,8 +78,7 @@ clean:
 	rm -f *~ $(TOP)/hu/generated.arm/* dbus/generated_*.* $(OBJS) $(APP) $(DEPS) $(APPLICATION_NAME)_*.zip
 
 install: all
-	cp -a -f headunit installer/config/androidauto/data_persist/dev/bin/
-
+	cp -a -f headunit installer/config/androidauto/data_persist/dev/bin/; cp -a -f headunit.json installer/config/androidauto/data_persist/dev/bin/
```

Comment:
this is unnecessary, while the wrapper generates the config file to /tmp/root :)

Reply:
That's true, but I didn't think of updates right now :) But I think there was an idea also, that AIO could generate the config for the user (or just sed the provided default config)